src/redux/store/index.ts

```diff
@@ -9,12 +9,10 @@ import {persistGlobalStore, persistRootReducer} from './persistance';
 
 const configureStore = (initialState?: DeepPartial<WebState>, onReady?: () => void) => {
   const root = persistRootReducer(combineReducers(rootReducer(history)));
-// @ts-ignore
```

Comment:
This used to break without ts-ignore, it doesn't any more?

Reply:
Nope, linter comes up clean and the app runs just fine. I can't find any issues.

Reply:
Well, it was working when I opened this PR. But now I've merged dev in and the linter is mad, so I'm not sure what changed. Either way I'll just put it back.